apps/src/templates/CollapserIcon.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+
+const collapseIconStyles = {
+  icon: {
+    fontSize: '18px'
+  }
+};
+
+function CollapserIcon({
+  isCollapsed,
+  onClick,
+  collapsedIconClass,
+  expandedIconClass,
+  style
+}) {
+  const iconClass = isCollapsed ? collapsedIconClass : expandedIconClass;
+
+  return (
+    <i
+      id="ui-test-collapser"
+      onClick={onClick}
+      role="button"
+      className={iconClass + ' fa'}
+      style={{...collapseIconStyles.icon, ...style}}
+    />
+  );
+}
+
+CollapserIcon.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  isCollapsed: PropTypes.bool.isRequired,
+  collapsedIconClass: PropTypes.string,
+  expandedIconClass: PropTypes.string,
+  style: PropTypes.object
+};
+
+CollapserIcon.defaultProps = {
+  collapsedIconClass: 'fa-chevron-circle-down',
+  expandedIconClass: 'fa-chevron-circle-up',
+  style: {}
+};
+
+export default Radium(CollapserIcon);
```

Comment:
imo we shouldn't use Radium if we don't need it. the main use case for Radium in our codebase is to allow hover effects, which you don't appear to be using. we have a few examples of components using it for its style merging functionality, but you're not using that here either. so i'd vote for removing it.

Reply:
Oops, this was a copy/pasta mistake